packages/hemera/lib/index.js

```diff
@@ -467,12 +467,13 @@ class Hemera extends EventEmitter {
    * Create a custom super error object in a running hemera instance
```

Comment:
You also can create errors without starting hemera. Rename it to
"Create a super error object. See https://github.com/busbud/super-error for more informations"